x/epochstorage/keeper/stake_storage.go

```diff
@@ -294,7 +294,7 @@ func (k Keeper) ModifyUnstakeEntry(ctx sdk.Context, stakeEntry types.StakeEntry)
 	if !found {
 		// should not happen since stake storage must always exist; do nothing to avoid panic
 		utils.LavaFormatError("critical: ModifyUnstakeEntry failed to get stakeStorage", legacyerrors.ErrNotFound,
-			utils.LogAttr("stakeAddr", stakeEntry.Address),
+			utils.LogAttr("stakeAddr", stakeEntry.Operator),
```

Comment:
I return the operator address in errors because it's the address that operates the provider process, so it'll make more sense to a user seeing the error (vault only holds funds)